enterprise/backend/src/metabase_enterprise/serialization/api.clj

```diff
@@ -160,8 +187,22 @@
   [:as {raw-params :params}]
   (api/check-superuser)
   (try
-    (let [{:keys [log-file callback]} (unpack&import (get-in raw-params ["file" :tempfile])
-                                                     (get-in raw-params ["file" :size]))]
+    (let [start              (System/currentTimeMillis)
+          {:keys [log-file
+                  error-message
+                  report
+                  callback]} (unpack&import (get-in raw-params ["file" :tempfile])
+                                            (get-in raw-params ["file" :size]))
+          imported           (set (map (comp :model last) (:seen report)))]
```

Comment:
Since you print out this map, it would be nice to use a `sorted-set` to make the string more stable and easy for humans to scan.

I also think `into` with a transducer is more idiomatic for this kind of transformation.

Reply:
done

Reply:
I don't see the changes 😕 

Not blocking, but maybe commit / push if you have it locally.